### ISSUE TITLE
🛠️  Add Ampstor as third-party story creation tool

### ISSUE THREAD

**Issue by marinatrajk** [ampproject/amp.dev#4978]
### Story creation resource name

Ampstor

### Type

Use [x] to mark a selection:

- [x ] Story Creation Tool
- [ ] CMS
- [ ] Platform
- [ ] Other

### URL

https://ampstor.co/

### Image

![editor](https://user-images.githubusercontent.com/9975218/99602651-b4db7500-29b6-11eb-9728-9975001067fc.png)

### Description

Visually create Web stories in a couple of minutes. Use pre-built templates, free media, and integrate with WordPress, Webflow, and other no-code tools.   

### List of Stories

< Please list 10 Web Stories across a diverse set of topics/treatments that each meet the following criteria

1. https://ampstor.app/story?id=DLbmIoIea
2. https://ampstor.app/story?id=HM7hj2xWh
3. https://ampstor.app/story?id=AvcnCbibp
4. https://ampstor.app/story?id=-Nq4DjApd
5. https://ampstor.app/story?id=_B18sQBgI
6. https://ampstor.app/story?id=XAnDhwkuL
7. https://ampstor.app/story?id=mYktrtNwh
8. https://ampstor.app/story?id=BELhYY-e_BOL 
9. https://ampstor.app/story?id=2XW61iXzd
10. https://ampstor.app/story?id=1aoPOiZHV

* Minimum of 7 pages
* At least 4 images
* At least 2 videos
* At least four pages with some text >

**Comment by shishirm** [comment]
Hello,

Thanks so much for filing this issue. Given this is Thanksgiving week and most folks are OOO, we'll start reviewing next week and will attempt to complete by the end of next week. We'll reach out with any clarifying questions or changes requested.

Thanks again!

**Comment by shishirm** [comment]
Hello,

Thanks again for submitting your tool to be listed on amp.dev. 

We're providing feedback in two categories. Must-Fix feedback which are things you will need to fix and we will verify before we can list the tool on amp.dev. Nice-To-Have/Suggestions which are recommendations on how to improve the Stories and the tools to provide the best user experience.

While the feedback is provided on the Stories themselves, the suggestion would be to implement features at the tool level such that they can be applied to all created Stories.

Must-Fix

- poster-portrait-src is not the right dimensions. These should be at least 640px by 853 px and should retain the 3:4 aspect ratio. You can find more details [here](https://amp.dev/documentation/components/amp-story/#poster-guidelines-(for-poster-portrait-src,-poster-landscape-src,-and-poster-square-src)).
- `<meta charset="utf-8" />` should be the first meta tag on the page
- The poster-portrait-src should be representative of the entire Story and right now its just a grey image. This is the image used to display the Story on various surfaces. Additionally, this should not contain any burned in text as the title of the Story may be overlaid on top.
- [This](https://ampstor.app/story?id=-Nq4DjApd) particular Story is failing AMP Validation as some videos do not have the required poster attribute set on them.

Nice-To-Have/Suggestions

- Provide descriptive alt text for images as per [here](https://blog.amp.dev/2020/02/12/seo-for-amp-stories/).
- Preload the AMP runtime as per [here](https://amp.dev/documentation/guides-and-tutorials/optimize-and-measure/optimize_amp/#optimize-the-amp-runtime-loading).
- Provide the ability to set schema.org style Structured Data for Stories as per [here](https://blog.amp.dev/2020/02/12/seo-for-amp-stories/).
- Some image / video edges are clipped or are obscured by text when Stories are viewed on different devices.
- If videos contain burned in text, provide the ability to set the exact copy of the text in the title attribute for the video element. Ideally provide the ability to set subtitles for videos with voice over.
- Some of the pages are heavy walls of text. The recommendation is to keep text on pages to <280 characters.
- On some pages, the text is hard to read with the background or moving videos. Providing the ability to add partial scrims can make the text more easily readable.

Please let us know once you've fixed the Must-Fix items and we will re-review and subsequently create a PR to list on amp.dev.

**Comment by patrickkettner** [comment]
@marinatrajk I am going to close this given the time without update, but please comment if you would like it reopened or have any questions

**Comment by marinatrajk** [comment]
Hey @patrickkettner we will start working on these issues now. Can we reopen this issue?
